### PR TITLE
feat: add fixture for quick dev data setup

### PR DESCRIPTION
**Pull request by alejandrosame**
Add a fixture file for the `shared` app to use during development. It contains a subset of ~100 CVEs and ~100 evaluation entries.

The README has been updated to include instructions on how to load and recreate the fixture file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,44 @@ This is the default Django workflow.
...
+manage flush
+
+# Ingest CVE data
+manage ingest_bulk_cve --test
+
+# Register a Nix Channel in the database
+src/website/manage.py register_channel <null> nixos-unstable UNSTABLE
+
+# Ingest evaluation data
+manage ingest_manual_evaluation d616185828194210bfa0e51980d78a8bcd1246cc nixos-unstable evaluation.jsonl
```
Comment (RaitoBezarius) [inline README.md:80]: Is this the real SHA1? Just to be certain.

**Review by RaitoBezarius — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (RaitoBezarius) [reply README.md:80]: It'd be nice if the SHA1 matches the true SHA1.

Comment (alejandrosame) [reply README.md:80]: For better context, I'm working on streamlining the setup of data for dev in [this PR](https://github.com/Nix-Security-WG/nix-security-tracker/pull/83) by introducing a `data_cache` directory. Next step will be to something similar for the evaluations ingestion, where the process will make always clear which hash generated the json file.

For the current README information, do you know which was the hash that produced the file at https://files.lahfa.xyz/private/evaluation.jsonl.zst.

Comment (RaitoBezarius) [reply README.md:80]: I believe it is `d616185828194210bfa0e51980d78a8bcd1246cc` according to my shell history logs.

File: README.md [hunk 2]
```diff
@@ -50,7 +50,44 @@ This is the default Django workflow.
...
+
+# Dump the database into the fixture file
+manage dumpdata shared > src/website/shared/fixtures/sample.json
+```
+
+These data ingestion steps are explained in greater detail in the `Manual ingestion` section below.
+
```
Comment (fricklerhandwerk) [inline README.md:87]:
```suggestion
```

**Pull request merged**